service_container.rst

```diff
@@ -138,85 +140,94 @@ it can't be re-used. Instead, you decide to create a new class::
         }
     }
 
-Congratulations! You've just created your first service class. Next, you can *teach*
-the service container *how* to instantiate it:
+Congratulations! You've just created your first service class! You can use it immediately
+inside your controller::
 
-.. _service-container-services-load-example:
+    use AppBundle\Service\MessageGenerator;
 
-.. configuration-block::
+    public function newAction(MessageGenerator $messageGenerator)
+    {
+        // thanks to the type-hint, the container will instantiate a
+        // new MessageGenerator and pass it to you!
+        // ...
 
-    .. code-block:: yaml
+        $message = $messageGenerator->getHappyMessage();
+        $this->addFlash('success', $message);
+        // ...
+    }
 
-        # app/config/services.yml
-        services:
-            # default configuration for services in *this* file
-            _defaults:
-                autowire: true
-                autoconfigure: true
-                public: false
+When you ask for the ``MessageGenerator`` service, the container constructs a new
+``MessageGenerator`` object and returns it (see sidebar below). But if you never ask
+for the service, it's *never* constructed: saving memory and speed. As a bonus, the
+``MessageGenerator`` service is only created *once*: the same instance is returned
+each time you ask for it.
 
-            # loads services from whatever directories you want (you can update this!)
-            AppBundle\:
-                resource: '../../src/AppBundle/{Service,Command,Form,EventSubscriber,Twig,Security}'
+.. _service-container-services-load-example:
 
-    .. code-block:: xml
+.. sidebar:: Automatic Service Loading in services.yml
 
-        <!-- app/config/services.xml -->
-        <?xml version="1.0" encoding="UTF-8" ?>
-        <container xmlns="http://symfony.com/schema/dic/services"
-            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xsi:schemaLocation="http://symfony.com/schema/dic/services
-                http://symfony.com/schema/dic/services/services-1.0.xsd">
+    The documentation assumes you're using
+    `Symfony Standard Edition (version 3.3) services.yml`_ configuration. The most
+    important part is this:
```

Comment:
As we're using actions with services as arguments almost everywhere, isn't the controller part important too?